src/main/java/httpserver/models/Response.java

```diff
@@ -10,8 +10,8 @@
 
     public Response() {}
 
-    public Response(String protocol, String statusCode, HashMap<String, String> headers, String body) {
-        this.protocol = protocol;
+    public Response(String statusCode, HashMap<String, String> headers, String body) {
+        this.protocol = "HTTP/1.1";
         this.statusCode = statusCode;
```

Comment:
Nice clean up on your commits.

Reply:
Can we make protocol a constant? Seems like an important /specific value.

Reply:
Yes, just changed this and I added the status codes as constants in a StatusCodes class